File: Arrays101/MergeSortedArray.java
package Arrays101;

import java.util.Arrays;

public class MergeSortedArray {
    public static void main(String[] args) {

    }

    public static void merge(int[] nums1, int m, int[] nums2, int n) {
        int[] temp = new int[m + n];
        int tempa = 0;

        if (nums1.length == 1 && nums2.length == 1) {
            nums1[0] = nums2[0];
        } else {
            for (int i = m; i < nums1.length; i++) {
                nums1[i] = nums2[tempa];
                tempa++;
            }
        }
        Arrays.sort(nums1);
    }
}

File: Arrays101/Inserting Item Into An Array/MergeSortedArray.py
from typing import List

def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
    if len(nums2) == 0:
        return nums1
    
    holder = m + n
    i = 0
    while i < len(nums2) - 1:
        nums2 [holder] = nums1[i]
        holder +=1
        i += 1

    nums1.sort()     